cloud.foundry.cli/src/main/java/cloud/foundry/cli/operations/ServicesOperations.java

```diff
@@ -45,252 +48,241 @@ public ServicesOperations(DefaultCloudFoundryOperations cloudFoundryOperations)
      * @return Mono object of all services as list of ServiceBeans
      */
     public Mono<Map<String, ServiceBean>> getAll() {
-
         return  this.cloudFoundryOperations.services()
-            .listInstances()
-            .flatMap(this::doGetServiceInstance)
-            .collectMap(ServiceInstance::getName, ServiceBean::new);
+                .listInstances()
+                .flatMap(serviceInstanceSummary -> getServiceInstance(serviceInstanceSummary.getName()))
+                .collectMap(ServiceInstance::getName, ServiceBean::new);
     }
 
-    private Mono<ServiceInstance> doGetServiceInstance(ServiceInstanceSummary serviceInstanceSummary) {
+    private Mono<ServiceInstance> getServiceInstance(String serviceName) {
         return this.cloudFoundryOperations
-            .services()
-            .getInstance(GetServiceInstanceRequest
-                .builder()
-                .name(serviceInstanceSummary.getName())
-                .build());
+                .services()
+                .getInstance(createGetServiceInstanceRequest(serviceName));
+    }
+
+    private GetServiceInstanceRequest createGetServiceInstanceRequest(String serviceName) {
+        return GetServiceInstanceRequest
+            .builder()
+            .name(serviceName)
+            .build();
     }
 
     /**
      * Creates a new service in the space and binds apps to it. In case of an error,
      * the creation- and binding-process is discontinued.
```

Comment:
The documentation does not fit the implementation. It should mention that a request is created. Also the discontinuation can be changed afterwards by manipulating the resulting mono. It would also be great to state, that the mono takes care of logging.